resources/wikia/modules/tracker.js

```diff
@@ -370,8 +370,14 @@
 			track.apply( null, args );
 		}
 
-		require(['wikia.trackingOptIn'], function (trackingOptIn) {
-			trackingOptIn.pushToUserConsentQueue(flushInternalTrackingQueue);
+		require([require.optional('wikia.trackingOptIn')], function (trackingOptIn) {
```

Comment:
Is this the only place we need to do this? What about `universal_analytics`?

Reply:
generally trackingOptIn is always available, it's only not there if the asset was somehow blocked (e.g. crazy antivirus). it's fine if GA fails in that case since that cannot block the page, but tracker calls waiting for callbacks can unfortunately cause such blockage that we need to alleviate